StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1000KeywordsMustBeSpacedCorrectly.cs

```diff
@@ -60,7 +60,7 @@ public override void Initialize(AnalysisContext context)
             context.RegisterSyntaxTreeAction(this.HandleSyntaxTree);
 
             // handle nameof (which appears as an invocation expression??)
-            context.RegisterSyntaxNodeAction(this.HandleInvocationExpressionSyntax, SyntaxKind.InvocationExpression);
+            context.RegisterSyntaxNodeActionHonorExclusions(this.HandleInvocationExpressionSyntax, SyntaxKind.InvocationExpression);
```

Comment:
:exclamation: We're also going to need to handle `RegisterSyntaxTreeAction` above.